frontend/src/pages/InsideStudyMode.js

```diff
@@ -0,0 +1,29 @@
+import React, { useState } from "react";
+import Header from "../main-components/Header";
+import Sidebar from "../main-components/Sidebar";
+import StudyModeContent from "../main-components/StudyModeContent";
+import css from "./template.css";
+
+function InsideStudyMode(props) {
+  // Handles mobile menu button and updates sidebar view
+  const [sideSetting, setSideSetting] = useState("f");
```

Comment:
Why can't these be true/false?

Reply:
Yeah, I can change it to true or false but its gonna have to happen in all my other files with sideBars. But I do think its worth changing.

Reply:
done.